x-pack/plugins/observability_solution/ux/public/application/ux_app.tsx

```diff
@@ -18,10 +18,7 @@ import { AppMountParameters, CoreStart, APP_WRAPPER_CLASS } from '@kbn/core/publ
 
 import { KibanaRenderContextProvider } from '@kbn/react-kibana-context-render';
 import { KibanaThemeProvider } from '@kbn/react-kibana-context-theme';
-import {
-  KibanaContextProvider,
-  useDarkMode,
-} from '@kbn/kibana-react-plugin/public';
+import { KibanaContextProvider, useDarkMode } from '@kbn/kibana-react-plugin/public';
```

Comment:
Did we need to do this for some recent EUI fix?  We should coordinate with o11y to see if they plan to fix this, potentially complicating a merge ...

Reply:
Oh no! i don't know how that got changed. I'll revert

Reply:
That was weird, the changes are gone now. Thanks!

Reply:
They came from this auto commit 9ec5e0c25e93c8c7396ea5ec02fee15090ac595c